### ISSUE TITLE
Compilation errors

### ISSUE THREAD

**Issue by ramsdell** [sslab-gatech/opensgx#1]
I cannot compile the software in the push two days ago on Ubuntu.  I have enclosed one fix. but I could not fix the many errors generated from the top-level make command.

John

diff --git a/user/polarssl/Makefile b/user/polarssl/Makefile
index ce780c5..1a99c9c 100755
--- a/user/polarssl/Makefile
+++ b/user/polarssl/Makefile
@@ -3,7 +3,7 @@ OBJS = rsa.o entropy.o ctr_drbg.o bignum.o md.o oid.o asn1parse.o sha1.o \
        md5.o ripemd160.o net.o aes_cmac128.o
 
 %.o: %.c
-	$(CC) -c -fPIC -Wall -o $@ $<
+	$(CC) -c -I../include -fPIC -Wall -o $@ $<
 
 all: $(OBJS)

**Comment by johnmwshih** [comment]
Did you still have this issue?

**Comment by ramsdell** [comment]
I just pulled and was able to compile everything.  There is still one issue remaining.  You have four derived files in your repository that should be deleted.  I made no modifications to the repo, yet after building the software, git status gives:

On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   qemu/roms/seabios/config.mak
	modified:   qemu/roms/vgabios/config.mak
	modified:   user/sgx-runtime
	modified:   user/sgx-test-runtime

These files conflict when I do a pull.  Furthermore, git clean does not remove them.

Thanks for your quick fix.
